jupyter-R.el

```diff
@@ -36,6 +41,15 @@ This fixes the header column alignment when the table has row names."
       `("" . ,(butlast header))
     header))
 
+(defun jupyter-org-result--R-html-list (parsed-html)
+  "Renders list-like html outputs.
+Since these can have arbitrary nested structure, we don't try to parse it into
```

Comment:
Can we not recursively parse out the list structure here? Or are you saying that there can be stuff in the lists that org syntax doesn't allow as being part of a list?

Reply:
I haven't extensively tested all the possibilities. But, for example, it's possible for it to return a list of tables, or list of lists of lists, etc. I'm not sure how well org can render these sorts of structures. It seemed simpler to just render the html with `shr.el` for now.

Reply:
OK, yeah I don't think org can handle all of that.